File: array_problems/findWrongNumber/findWrongNumber.py
def findWrongNUmber(arr):
    for i in range(1, len(arr)):
        if arr[i] != arr[i - 1] + 1:
            return arr[i]
    return None


response = findWrongNUmber([1, 2, 3, 4, 17, 5, 6, 7, 8])
print(response)

File: array_problems/findWrongNumber/findWrongNumber.php
<?php

function findWrongNumber($arr) {
    for ($i = 0; $i < count($arr); $i++) {
        if($i > 0 && $arr[$i] !== $arr[$i - 1] + 1) return $arr[$i];
    }
}
$response = findWrongNumber([1, 2, 3, 4, 17, 5, 6, 7, 8]);
echo($response);